templates/e-commerce/partials/hero/hero.snip.html

```diff
@@ -0,0 +1,20 @@
+{{#hero}}
+  <div class="e-commerce-ampstart-hero center">
+    <div class="e-commerce-ampstart-hero-img">
+      {{#images}}
+        {{#image}}
+          <amp-img src="{{url}}" width="{{width}}" height="{{height}}" layout="responsive" media="{{media}}" alt="{{alt}}"></amp-img>
+        {{/image}}
+      {{/images}}
+    </div>
+    <div class="e-commerce-ampstart-hero-content e-commerce-ampstart-hero-theme-{{text-theme}} flex justify-center flex-column">
+      <div>
```

Comment:
I added this div to ensure that flexbox only vertically centers the contents of this element as a block. Removing this results in the hero content being displayed incorrectly. I couldn't find any basscss classes to help.